src/main.cc

```diff
@@ -117,7 +135,12 @@ auto jsonschema_main(const std::string &program, const std::string &command,
               << sourcemeta::jsonschema::cli::PROJECT_VERSION << "\n";
     std::cout << "Usage: " << std::filesystem::path{program}.filename().string()
               << " <command> [arguments...]\n";
-    std::cout << USAGE_DETAILS;
+    if (use_colors) {
+      std::cout << termcolor::yellow << USAGE_DETAILS << termcolor::reset
```

Comment:
I don't think it makes much sense to color out the entire help. Maybe just color out the project name in line 134?